scene/main/node.cpp

```diff
@@ -135,15 +135,15 @@ void Node::_notification(int p_notification) {
 
 		case NOTIFICATION_ENTER_TREE: {
 			ERR_FAIL_NULL(get_viewport());
-			ERR_FAIL_NULL(get_tree());
+			ERR_FAIL_NULL(data.tree);
 
-			if (get_tree()->is_accessibility_supported() && !is_part_of_edited_scene()) {
-				get_tree()->_accessibility_force_update();
-				get_tree()->_accessibility_notify_change(this);
+			if (data.tree->is_accessibility_supported() && !is_part_of_edited_scene()) {
+				data.tree->_accessibility_force_update();
+				data.tree->_accessibility_notify_change(this);
```

Comment:
Are these changes required for this PR?
The wrapper function is still available right?

Reply:
Yeah it's not required, but it's somewhat related, so I included it.
I can move it to a separate commit or PR if that's preferred.

Reply:
Yeah I agree on the check in `get_tree()`, I have always thought this.

I wonder whether it is there due to binding and making error reporting easier from gdscript, there seem a lot of cases like this where the form is:

* does it exist?
* get it

Instead of:
* get it
* check NULL